crates/forge-runner/src/running/setup.rs

```diff
@@ -40,7 +40,19 @@ pub fn initialize_execution_context<'a>(
     // Instantiate Cairo runner.
     let proof_mode = false;
     let trace_enabled = true;
-    let mut runner = CairoRunner::new(program, LayoutName::all_cairo, proof_mode, trace_enabled)?;
+    let mut runner = CairoRunner::new(
+        program,
+        LayoutName::all_cairo,
+        // The `dynamic_layout_params` argument should only be used with dynamic layout.
+        // It is ignored otherwise.
+        // See: https://github.com/lambdaclass/cairo-vm/blob/4f02f5aab30f9062c2db1f5addbf28d34843168e/vm/src/vm/runners/cairo_runner.rs#L171
+        None,
+        proof_mode,
+        trace_enabled,
+        // By default, padding should be enabled
+        // See: https://github.com/lambdaclass/cairo-vm/blob/e339ab7cea2794428cff791a84c45841b84ca854/vm/src/cairo_run.rs#L37-L44
+        false,
```

Comment:
https://github.com/lambdaclass/cairo-vm/blob/4f02f5aab30f9062c2db1f5addbf28d34843168e/vm/src/vm/runners/cairo_runner.rs#L233

Seems that it can only be true in `proof_mode` which is false for us so it's good. 
But it could be worth verifying what these paddings are doing to the trace beyond the comment.